internal/utils/csi.go

```diff
@@ -331,10 +366,15 @@ var TopologyContainerArg = "--feature-gates=Topology=true"
 var RecoverVolumeExpansionFailureContainerArg = "--feature-gates=RecoverVolumeExpansionFailure=true"
 var EnableVolumeGroupSnapshotsContainerArg = "--enable-volume-group-snapshots=true"
 var ForceCephKernelClientContainerArg = "--forcecephkernelclient=true"
+var LogToFile = "--logtostderr=false"
+var LogToStderr = "--alsologtostderr=true"
 
 func LogLevelContainerArg(level int) string {
 	return fmt.Sprintf("--v=%d", Clamp(level, 0, 5))
 }
+func LogRotationSpecArg(LogHostPath, CsiComponentName, containerName string) string {
```

Comment:
Name should be `LogFileContainerArg` to match the arg name and the `ContainerArg` suffix